src/GovUk.Education.ExploreEducationStatistics.Common/BlobContainers.cs

```diff
@@ -10,6 +10,7 @@ public static class BlobContainers
     {
         public static readonly IBlobContainer PrivateReleaseFiles = new BlobContainer("releases");
         public static readonly IBlobContainer PublicReleaseFiles = new BlobContainer("downloads");
+        public static readonly IBlobContainer PrivateContent = new PrivateBlobContainer("cache");
         public static readonly IBlobContainer PublicContent = new BlobContainer("cache");
```

Comment:
I don't think it will make a difference but we could make `PublicContent` a type of `PublicBlobContainer` so it gets the name `public-cache` locally now that we have `private-cache`, similar to how we have `public-methodologies` and `private-methodologies`.

Reply:
I think this would be good - just conscious that time is ticking on this expedite, and this would require a bit of manual migration as part of rollout.  I could look at doing this as part of the lower priority caching of Subject Meta in Admin, which is a follow-up ticket to this?

Reply:
I was just thinking it should only affect local development using the storage emulator as non-emulated environments will use "cache" for both public and private which exist in totally separate storage accounts.

With the current setup, locally emulated containers will both be in the same storage account (there's only one anyway) and will be named "private-cache" and "cache" but could be "private-cache" and "public-cache" just for consistency.

Fine to do it later, or even just test what I say here later is correct 🙂 .

Reply:
Oh I see, no that sounds like a good idea then!

Reply:
Done now thanks.